migration-approaches/Curated-List.md

```diff
@@ -0,0 +1,7 @@
+# Curated list of community migration approaches
+
+List of interesting approaches which should help to migrate. It might be an [architectural approach](#architectural-approaches) or just tips and tricks / experiences to keep in mind.
+
+## Architectural approaches
+
+* [Leveraging Clean Architecture for Migration](https://github.com/lwluc/camunda-ddd-and-clean-architecture/pull/1)
```

Comment:
Could you add a sentence or two about the context and your special approach to migration?

Reply:
Hey @ingorichtsmeier I added a short description. I hope it introduces the approach well. What do you think?

Reply:
Looks good!